### PR TITLE
feat(constellation): update to latest version that supports nonce

### PR DESCRIPTION
**Pull request by schnogz**

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/Trade.tsx [hunk 1]
```diff
@@ -159,7 +179,10 @@ const Trade = ({
         <FormattedMessage id='modals.trade.header' defaultMessage='Shortcuts' />
       </HeaderText>
       <IconWrapper onClick={handleClose}>
-        <Icon name='close' color='grey600' role='button' data-e2e='close' size='24px' cursor />
+        {/* @ts-ignore */}
+        <HoverIcon data-e2e='close' label='arrow up' size='md' color='grey600' role='button'>
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Flyout/Brokerage/Trade.tsx:183]: @pedroapfilho @awiermanncasas-bc 
1) should we have built in support for Icons to have a `cursor: pointer` attribute?  
2) typescript doesnt like the `role` attribute being added here. something we can fix?

File: packages/blockchain-wallet-v4-frontend/src/components/Form/TextInputWithClipboard/index.tsx [hunk 2]
```diff
@@ -35,9 +35,9 @@ const TextInputWithClipboard: React.FC<Props> = ({ value }) => {
   return (
     <Wrapper>
       <StyledInput disabled value={value} />
-      <IconWrapper>
-        <CopyToClipboard color='blue400' value={value} />
-      </IconWrapper>
+      {/* <IconWrapper> */}
+      {/*  <CopyToClipboard color='blue400' value={value} /> */}
+      {/* </IconWrapper> */}
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/TextInputWithClipboard/index.tsx:40]: @pedroapfilho @awiermanncasas-bc 
1) can one of you update this to use the copy for clipboard hook?

File: packages/blockchain-wallet-v4-frontend/src/components/Navbar/MobileDropdown/template.tsx [hunk 3]
```diff
@@ -62,12 +63,16 @@ const MobileDropdown = () => {
 
   return (
     <NavbarButton data-e2e='mobileQrCode' ref={ref}>
-      <Icon color={colors.grey400} name={IconName.PHONE} size='sm' onClick={handleMenuToggle} />
+      {/* @ts-ignore */}
+      <Icon label='phone' size='sm' color='grey400' onClick={handleMenuToggle}>
+        <IconPhone />
+      </Icon>
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Navbar/MobileDropdown/template.tsx:69]: @pedroapfilho @awiermanncasas-bc 
1) Icon component and TS dont like the `onClick` attribute. can we fix?

**Review by pedroapfilho — APPROVED** [review]

**Comment by awiermanncasas-bc** [comment]
@schnogz is this PR ready, or does it still have to update the copy component and the mobile card?

Screen Shot 2022-04-13 at 3 38 52 PM

Because I have a branch where I fixed some of these issues.
I will spend some time to rebase that with development and see if that still works

**Pull request closed without merging**